### PR TITLE
[Workers KV] Create Reference section and move content

### PR DESCRIPTION
**Pull request by Maddy-Cloudflare**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -55,4 +55,4 @@ tutorial_token = "xxa5a7d4c9-db72-424d-860d-ae68ff7f60ac"
 
 [caches]
   [caches.getjson]
-    maxAge = "24h"
+    maxAge = "120h"
```
Comment (marciocloudflare) [inline config.toml]: maybe remove this, as Kody already made a PR that has this in?

File: package.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
     "typescript": "~4.7.4",
     "vite": "^4.0.0",
     "vue-tsc": "^1.0.12",
-    "wrangler": "^3.1.1"
+    "wrangler": "^3.22.4"
```
Comment (marciocloudflare) [inline package.json:46]: Any reason to change this?

Comment (marciocloudflare) [inline package.json]: Any reason to change this is the package.json file?

**Review by marciocloudflare — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (Maddy-Cloudflare) [reply package.json]: I'll revert this, I was trying to solve the GitHub error and forgot to revert this back.

**Pull request merged**